GameController.js

```diff
@@ -1,91 +1,115 @@
-class GameController {
-
-	createEnemy() {
-		snakeLiving = assetManager.getSprite("assets");
-		snakeLiving.regX = 50;
-		snakeLiving.regY = 50;
-		snakeLiving.x = enemyXPos;
-		snakeLiving.y = enemyYPos;
-		snakeLiving.gotoAndPlay("snakeAlive");
-		stage.addChild(snakeLiving);
-		console.log("loading");
+function GameController() { 
+	console.log(this);
+  // Set the position of the target
+  this.enemyXPos = 100;
+  this.enemyYPos = 100;
+
+  // Set the speed of the target
+  this.enemyXSpeed = 1.5;
+  this.enemyYSpeed = 1.75;
+
+  // This should be set either as constants in Game and passed, or refactored
+  // so all the code can be here.
+  this.canvasWidth  = 600;
+  this.canvasHeight = 600;
+};
+
+GameController.prototype.createEnemy = function() {
+	var snakeLiving = assetManager.getSprite("assets");
+	console.log(snakeLiving);
+	snakeLiving.regX = 50;
+	snakeLiving.regY = 50;
+	snakeLiving.x = this.enemyXPos;
+	snakeLiving.y = this.enemyYPos;
+	snakeLiving.gotoAndPlay("snakeAlive");
+	stage.addChild(snakeLiving);
+	console.log("loading");
+}
+
+GameController.prototype.handleMouseDown = function(event) {
```

Comment:
I would Google what `prototype` functions are in JS. They're cool. Essentially, the memory doesn't need to be saved in every instance if it's on the prototype, and it can still reference the variables and properties with `this.x` `this.y` etc.